README.md

```diff
@@ -67,6 +67,7 @@ In addition to the deployment options you must also configure the following.
 | ------------- | ------------- | ------------- | ------------- |
 | `GIT_CONFIG_NAME`  | Allows you to customize the name that is attached to the GitHub config which is used when pushing the deployment commits. If this is not included it will use the name in the GitHub context, followed by the name of the action.  | `with` | **No** |
 | `GIT_CONFIG_EMAIL`  | Allows you to customize the email that is attached to the GitHub config which is used when pushing the deployment commits. If this is not included it will use the email in the GitHub context, followed by a generic noreply GitHub email.  | `with` | **No** |
+| `REMOTE_REPOSITORY`  | If you'd like to push to an alternate repository, specify the `organization/repository` path.  | `with` | **No** |
```

Comment:
There was a `REPOSITORY_NAME` option added in `3.3.1` which will let you specify a different repository path if you update from `dev`.